test/integration/admin_can_modify_its_account_test.rb

```diff
@@ -0,0 +1,31 @@
+require 'test_helper'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.